website/conferences/views.py

```diff
@@ -188,16 +188,104 @@ def conference_data(meeting):
     except Conference.DoesNotExist:
         raise HTTPError(httplib.NOT_FOUND)
 
-    nodes = AbstractNode.objects.filter(tags__id__in=Tag.objects.filter(name__iexact=meeting, system=False).values_list('id', flat=True), is_public=True, is_deleted=False)
-    ret = []
-    for idx, each in enumerate(nodes):
-        # To handle OSF-8864 where projects with no users caused meetings to be unable to resolve
-        try:
-            ret.append(_render_conference_node(each, idx, conf))
-        except IndexError:
-            sentry.log_exception()
-    return ret
+    return conference_submissions_sql(conf)
+
+def conference_submissions_sql(conf):
+    """
+    Serializes all meeting submissions to a conference (returns array of dictionaries)
+
+    :param obj conf: Conference object.
 
+    """
+    submission1_name = conf.field_names['submission1']
+    submission2_name = conf.field_names['submission2']
+    conference_url = web_url_for('conference_results', meeting=conf.endpoint)
+    abstract_node_content_type_id = ContentType.objects.get_for_model(AbstractNode).id
+    osf_user_content_type_id = ContentType.objects.get_for_model(OSFUser).id
+
+    with connection.cursor() as cursor:
+        cursor.execute(
+            """
+            SELECT  json_build_object(
+                    'id', osf_abstractnode.id,
+                    'title', osf_abstractnode.title,
+                    'nodeUrl', '/' || GUID._id || '/',
+                    'author', CASE WHEN AUTHOR.family_name != '' THEN AUTHOR.family_name ELSE AUTHOR.fullname END,
+                    'authorUrl', '/' || AUTHOR_GUID._id || '/',
+                    'category', COALESCE(MEETING_CATEGORY.name, %s),
+                    'download', COALESCE(DOWNLOAD_COUNT, 0),
+                    'downloadUrl', COALESCE('/project/' || GUID._id || '/files/osfstorage/' || FILE._id || '/?action=download', ''),
+                    'dateCreated', osf_abstractnode.created,
+                    'confName', %s,
+                    'confUrl', %s,
+                    'tags', TAGS_LIST.tag_list
```

Comment:
Do we actually use `tags` on the frontend? I don't see it in the UI anywhere. Maybe we can remove this?

Reply:
Tags are included because filtering also searches tags.